vqe.py

```diff
@@ -0,0 +1,84 @@
+import networkx as nx
```

Comment:
Please add docstring at the start of the example file, following sphinx-gallery, and describe the problem we are going to look at. State the hamiltonian (use math env) as well as what model and physical system it corresponds to. 

see for example https://github.com/TeamGraphix/graphix/blob/master/examples/qft.py and https://sphinx-gallery.github.io/stable/syntax.html for how to write rst-style docstring in python file. Render this example locally, by running `make html` in `docs` folder (check `requirements.txt` in docs folder so you can run sphinx-gallery.